packages/augur-ui/src/modules/create-market/components/form.tsx

```diff
@@ -464,6 +466,9 @@ export default class Form extends React.Component<FormProps, FormState> {
           description: 'Invalid',
           isTradable: true,
         });
+      } else if (newMarket.marketType === SCALAR) {
+        outcomesFormatted = SCALAR_OUTCOMES;
+        outcomesFormatted[1].description = newMarket.scalarDenomination === "" ? NON_EXISTENT : newMarket.scalarDenomination;
```

Comment:
in market creation scalar denomination isn't allowed to be null?

Reply:
they might not have entered the scalar denomination yet at this point

Reply:
ok, works for me